CMakePresets.json

```diff
@@ -73,17 +73,9 @@
                     "type": "BOOL",
                     "value": false
                 },
-                "PNG_LIBRARY": {
```

Comment:
Please don't take it out of the preset. The whole idea is that if you need a different preset you create a different preset. I've my own variation of this preset for my wsl builds and this would break it.